src/aero_data.F90

```diff
@@ -332,6 +356,28 @@ subroutine aero_data_set_mosaic_map(aero_data)
 
   end subroutine aero_data_set_mosaic_map
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+  !> Get the index on the CAMP state array for a specified species and
+  !! computational particle
+  integer function camp_spec_id(aero_data, i_part, i_spec)
```

Comment:
@jcurtis2 when I turned on the extra warnings, this function broke the non-CAMP test build. Can you figure out how to fix it?

Reply:
Fixed this by returning a value of 0 if this code is compiled without CAMP support.